plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java

```diff
@@ -366,22 +355,19 @@ static <T extends AreaContainmentObject> void register(Class<T> type,  ObjectTag
         // -->
         processor.registerTag(ListTag.class, "approximate_overlap_areas", (attribute, area) -> {
             ListTag list = new ListTag();
-            CuboidTag boundary = area.getCuboidBoundary();
-            CuboidTag.LocationPair pair = boundary.pairs.get(0);
+            CuboidTag.LocationPair pair = area.getCuboidBoundary().pairs.get(0);
             NotedAreaTracker.forEachAreaThatIntersects(pair.low, pair.high, list::addObject);
             return list;
         });
     }
 
     default boolean areaBaseAdvancedMatches(String matcher) {
-        String matcherLow = CoreUtilities.toLowerCase(matcher);
-        if (matcherLow.startsWith("area_flagged:")) {

```

Comment:
Should have an `instanceof` here rather than changing the base type. An AreaContainmentObject doesn't _necessarily have to_ be flaggable

Reply:
Should be done